pkg/controllers/kube-apiserver.go

```diff
@@ -133,7 +133,7 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 			"kubelet-certificate-authority":   {cryptomaterial.CABundlePath(kubeCSRSignerDir)},
 			"kubelet-client-certificate":      {cryptomaterial.ClientCertPath(kubeletClientDir)},
 			"kubelet-client-key":              {cryptomaterial.ClientKeyPath(kubeletClientDir)},
-			"kubelet-preferred-address-types": {"Hostname"},
+			"kubelet-preferred-address-types": {"ExternalIP", "InternalIP", "ExternalDNS", "InternalDNS", "Hostname"},
```

Comment:
In microshift, only 'InternalIP' and 'Hostname' are provided by kubelet. Shall we remove the address types we don't use?

Reply:
In order to allow the CNCF conformance tests requiring multinode I dropped there the external counterparts, just in case.

Reply:
Wait, let me simplify it because you are right.

Reply:
Done. This also addresses your comment @dhellmann.
```bash
$ KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig oc get node -o jsonpath='{.items[*].status.addresses}' | jq
[
  {
    "address": "192.168.122.215",
    "type": "InternalIP"
  },
  {
    "address": "microshift-dev.local",
    "type": "Hostname"
  }
]
```